frontend/packages/dev-console/integration-tests/features/addFlow/create-from-git.feature

```diff
@@ -203,3 +203,13 @@ Feature: Create Application from git form
                   | git_url                                |
                   | https://github.com/sclorg/httpd-ex.git |
                   | https://github.com/sclorg/nginx-ex.git |
+
+        @regression @manual
+        Scenario: Provide custom build environments for nodejs git import
+            Given user is at Import from Git form
+            When user enters Git Repo URL as "https://github.com/sclorg/nodejs-ex"
+            And user enters "NPM_RUN" build environment field as "build2"
+            And user enters Name as "nodejs-env"
+            And user clicks Create button on Add page
+            Then user will be redirected to Topology page
+            And user is able to see environment variable "NPM_RUN" in details page for Build Config "nodejs-env"
```

Comment:
I'm sure the user needs to click on "Build #1" first, switch then to the environment variable tab and then can verify that the NPM_RUN variable is set.

Reply:
Updated to click on build instead of BuildConfig and changed statement text to reflect it. Or were you suggesting we do this in multiple separate steps instead of one?